perfscale_regression_ci/scripts/common.sh

```diff
@@ -60,4 +60,33 @@ function check_no_error_pods()
     #loop to find logs of error pods?
     exit 1
   fi
+}
+
+function wait_for_running() {
```

Comment:
I think we can take this function out of the scripts because kube-burner run_workload will cover this